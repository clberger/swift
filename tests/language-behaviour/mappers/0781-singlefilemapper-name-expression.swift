type messagefile;

(messagefile t) write() { 
    app {
        echo @filename(t) stdout=@filename(t);
    }
}

string fn = "0781-singlefilemapper-name-expression";

messagefile outfile <single_file_mapper;file=@strcat(fn,".","out")>;

outfile = write();

