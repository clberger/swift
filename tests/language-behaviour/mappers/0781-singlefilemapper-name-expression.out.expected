0781-singlefilemapper-name-expression.out
